Markdown/Markdown.sublime-syntax

```diff
@@ -98,21 +98,32 @@ variables:
           [ ]*
           ([ ]{0,3})   # up to 3 spaces
           (
-            `{3,}      #   3 or more backticks
+            (`){3,}    #   3 or more backticks
```

Comment:
Why did you add capture groups here? It's usually not a good idea to have those in variables because they affect the index for the captures map, and indeed you had to increase the offset of many captures from 3  to 5.

Reply:
I need these new capture groups in the `code_fence_escape` part because markdown blocks that are opened by *n* backticks/tildes can be closed by *n+x* respective closing characters, e.g:
```
~~~~    ruby startline=3 $%@#$
def foo(x)
  return 3
end
~~~~~~~
```

Reply:
Okay, but wasn't that functionality there beforehand? In this pr you just added capture groups but you didn't add any references to them (besides the scope association). Besides, the added groups only match a single character and not the while opening sequence.